File: TheMovie/app/src/main/java/com/nyinyi/nw/themovie/mvp/view/SearchView.java
package com.nyinyi.nw.themovie.mvp.view;

import com.nyinyi.nw.themovie.vos.UpcomingVO;

import java.util.List;

/**
 * Created by User on 9/20/2017.
 */

public interface SearchView {
    void onSearchSuccess(List<UpcomingVO> movieList);
}

File: TheMovie/app/src/main/java/com/nyinyi/nw/themovie/mvp/view/SearchView.kt
package com.nyinyi.nw.themovie.mvp.view

import com.nyinyi.nw.themovie.vos.UpcomingVO

/**
 * Created by User on 9/20/2017.
 */

interface SearchView {
    fun onSearchSuccess(movieList: List<UpcomingVO>)
}
